CONTRIBUTING.md

```diff
@@ -18,13 +18,16 @@ Contributions to this product are [released][releases] to the public under the [
 Please note that this project is released with a [Contributor Code of Conduct][code-of-conduct]. By participating in this project you agree to abide by its terms.
 
 ## Getting Started
+
+Please ensure that you have [homebrew](https://brew.sh/) installed. Instructions for setting this application up on an OS other than OSX are currently not outlined.
+
 This app is written in [ES6 JavaScript](https://nodejs.org/en/docs/es6/) and runs on [Node.js](https://nodejs.org/). After cloning the repository, install the dependencies by running:
 
 ```
 $ script/bootstrap
 ```
 
-This will install `node` and `postgres`, which are required to run the app. You will need to start up the postgres instance, and then run:
+This will install `node` and `mysql`, which are required to run the app. You will need to start up the postgres instance, and then run:
```

Comment:
Second postgres in there

Reply:
we don't use mysql at all, I've lost my dang mind.